docs/admin/authorization.md

```diff
@@ -99,21 +111,35 @@ A request has attributes which correspond to the properties of a policy object.
 When a request is received, the attributes are determined.  Unknown attributes
 are set to the zero value of its type (e.g. empty string, 0, false).
 
-An unset property will match any value of the corresponding
-attribute.  An unset attribute will match any value of the corresponding property.
+A property set to "*" will match any value of the corresponding attribute.
 
 The tuple of attributes is checked for a match against every policy in the policy file.
 If at least one line matches the request attributes, then the request is authorized (but may fail later validation).
 
-To permit any user to do something, write a policy with the user property unset.
-To permit an action Policy with an unset namespace applies regardless of namespace.
+To permit any user to do something, write a policy with the user property set to "*".
+To permit a user to do anything, write a policy with the namespace, resource, and nonResourcePath properties set to "*".
```

Comment:
all existing files will continue to work exactly as they do today. tests ensure "v0" rules are converted and behave identically